hugo/layouts/partials/copy_to_clipboard.html

```diff
@@ -0,0 +1,4 @@
+{{ if (findRE "<pre" .Content 1) }}
```

Comment:
For an alternative approach to detect copy-blocks, see [render hook examples](https://gohugo.io/render-hooks/code-blocks/#examples). The idea is to set `$.Page.Store.Set "needs_clipboard" true` in the shortcode template, and then use `{{ if $.Page.Store.Get "needs_clipboard" }}` Not 100% sure if what works for render hooks will work for shortcodes though.